src/Http/Cookie/CookieCollection.php

```diff
@@ -103,6 +106,22 @@ public function get($name)
     }
 
     /**
+     * Backwards compatibility helper for consumers of Client\CookieCollection
+     *
+     * @param string $url The url to get cookies for.
+     * @return array An array of matching cookies.
+     * @deprecated 3.5.0 Will be removed in 4.0.0. Use addToRequest() instead.
```

Comment:
anyway this method (` protected function getByUrl()`) is with protected scope and no need to this line